examples/evm-to-substrate-react-example/src/config/local.ts

```diff
@@ -0,0 +1,86 @@
+import { EvmBridgeSetupList } from "@buildwithsygma/sygma-sdk-core";
+import { SubstrateConfigType } from "./types";
+
+export const substrateConfig: SubstrateConfigType = {
+  domainId: "3",
+  appName: "sygma-sdk-substrate-react-example",
+  provider_socket: "ws://127.0.0.1:9944",
+  assets: [
+    {
+      assetName: "USDC",
+      assetId: 2000,
+      xsmMultiAssetId: {
+        concrete: {
+          parents: 1,
+          interior: {
+            x3: [
+              { parachain: 2004 },
+              { generalKey: "0x7379676d61" },
+              { generalKey: "0x75736463" },
+            ],
+          },
+        },
+      },
+    },
+  ],
+};
+export const evmSetupList: EvmBridgeSetupList = [
```

Comment:
Should we use the shared-config format here? In this case `feeHandlers` will have the fee strategies. If not, is the type updated? Should we also fill the `feeRouter` address?

Reply:
i've created #234 and #235 issues for that

Reply:
ok